cmds/webboot/utils.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+// WriteCounter counts the number of bytes written to it. It implements to the io.Writer
```

Comment:
Could you move the progress bar change to another PR? Lets try and keep each PR focused on 1 thing.

Reply:
Ok sure